cdbot/cogs/fun.py

```diff
@@ -151,6 +152,13 @@ async def on_ready(self):
 
         await self.migrate_quotes()
 
+    @Cog.listener()
+    async def on_member_update(self, before, after):
+        if before.status == Status.offline and after.status == Status.online:
+            if any(i.id == STAFF_ROLE_ID for i in after.roles):
+                alert_channel = self.bot.get_channel(LOGGING_CHANNEL_ID)
+                await alert_channel.send("A CD staff member came online :eyes:")
```

Comment:
It might be useful to have the message say which staff member came online i.e include the username of who came online.

Reply:
This functionality has been added.